microprofile/WFLY-13048_upgrade_microprofile-health_2.2.adoc

```diff
@@ -0,0 +1,61 @@
+= Upgrade MicroProfile Health to 2.2
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 3.3 provides a minor upgrade for https://github.com/eclipse/microprofile-health/releases/tag/2.2[MicroProfile Health 2.2] whose changes are documented in its https://download.eclipse.org/microprofile/microprofile-health-2.2/microprofile-health-spec.html#release_notes_2_2[release notes].
+
+This component upgrade also covers the upgrade of smallrye-health that implements MicroProfile Health 2.2.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-13048[WFLY-13048]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1426[EAP7-1426]
+* Support of the `mp.health.disable-default-procedures` config property is handled separately by to https://issues.jboss.org/browse/WFLY-12952[WFLY-12952] that adds such a default server procedure for the readiness check.
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+[X] QE
+
+== Requirements
+
+* There is no change in the requirements compared to the integration of MicroProfile Health 2.1.
+
+=== Non-Requirements
+
+* Health checks in subdeployments of EAR deployment are not supported.
+
+== Implementation Plan
+
+* Upgrade MicroProfile Health to 2.2
+* Upgrade smallrye-health to the version that implements MicroProfile Health 2.2
+
+== Test Plan
+
+* Run any MicroProfile Health test in WildFly integration test suite
+  * no new tests are required for MicroProfile Health 2.2
+* Run the Eclipse MicroProfile Health TCK without failures
```

Comment:
Can you please provide link to the MP Health TCK?

Reply:
What do you want? the [code base of the MP Health TCK](https://github.com/eclipse/microprofile-health/tree/master/tck) or the [module in WildFly](https://github.com/wildfly/wildfly/tree/master/testsuite/integration/microprofile-tck/health)

Reply:
My concern was about the first one, actually, as it something "external" to Wildfly project. maybe a link pointing to both of them would ease direct access to resources.

Reply:
done